Directory.Build.props

```diff
@@ -2,7 +2,8 @@
     <!-- C# analyzers configuration -->
     <PropertyGroup>
         <CodeAnalysisRuleSet>$(SolutionDir)global.ruleset</CodeAnalysisRuleSet>
-        <TargetFramework>net5.0</TargetFramework>
+        <TargetFramework>net6.0</TargetFramework>
+        <SuppressTrimAnalysisWarnings>true</SuppressTrimAnalysisWarnings>
```

Comment:
Suppressing trimmer analysis warnings might seem a bit foolish, but I'm not sure littering the code with tons of attributes to suppress them is really the way forward for us. Let's for now assume the "dynamic programming language" approach instead - _let's catch errors with unit tests_ instead of static analysis. :zany_face:

Time will tell whether this is insane or simply a stroke of genius...